### ISSUE TITLE
Can't find installer bundle

### ISSUE THREAD

**Issue by lukpueh** [SeattleTestbed/custominstallerbuilder#15]
I wonder if the CIB installation docs are misleading. When I setup the CIB django app as suggested I get the following error when I try to download the installer:
``` python
[Errno 2] No such file or directory: '/home/cib/custominstallerbuilder/html/static/installers/base/seattle_linux.tgz'
```
As far as I understand the custom installer bundle is created by `common/packager.py` which builds/packages it from the base installer at `BASE_INSTALLER_ROOT` -- a default setting found in `settings_base.py` that points to the directory seen in the error message above.

The docs [don't mention](https://github.com/lukpueh/docs/blob/issue4/Operating/CustomInstallerBuilder/Configuration.md#create-and-configure-local-settings) to change this setting. But they do suggest to install the base installers at `/var/www/dist`  [this section](https://github.com/lukpueh/docs/blob/issue4/Operating/CustomInstallerBuilder/Configuration.md#adapt-the-build-scripts).

Am I missing something?

**Comment by aaaaalbert** [comment]
You don't seem to be missing anything (apart from the correct information at the correct place, of course!)

`/var/www/dist` is indeed the "magic" / historic / agreed upon home directory for base installers. They'll actually end up below that in `installers/base_installers` IIRC, and the generated installers are cached in `installers/RANDOM_HASH_ID`.

In general you are free to choose whatever base installer dir. You are right that the CIB docs page should mention this, and so should the [infrastructure architecture page](https://github.com/SeattleTestbed/docs/blob/master/UnderstandingSeattle/SeattleInfrastructureArchitecture.wiki).

**Comment by aaaaalbert** [comment]
Speaking with @lukpueh we arrived at the conclusion that the underlying problem is a synchronization issue between documentation and implementation currently. The docs assume "the old way" of building installers that used a specific build shellscript. Current and future CIBs however use @XuefengHuang's Python port of this script which internally assumes a certain dir layout [pre-generated by the new Seattle buildscripts](https://github.com/SeattleTestbed/installer-packaging/blob/master/rebuild_base_installers.py#L13-L16) in order to work.

In other words, the CIB docs must be adapted to a significant degree. See SeattleTestbed/docs#15.
